src/main/java/mil/dds/anet/database/PersonDao.java

```diff
@@ -411,6 +411,13 @@ public int mergePeople(Person winner, Person loser) {
         .thenApply(l -> PersonPositionHistory.getDerivedHistory(l));
   }
 
+  public CompletableFuture<List<PersonPositionHistory>> getAllPositionHistory(
```

Comment:
This method is only (indirectly) called by the tests, so really belongs somewhere under `src/test/java`.

Reply:
This method is moved under `src/test/java`.